drivers/include/cc110x.h

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2014 Freie Universität Berlin
+ *               2015 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    drivers_cc110x CC110x
+ * @ingroup     drivers_cc110x
+ * @{
+ * @file
+ * @brief       Public interface for cc110x driver
+ * @author      Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#ifndef CC110X_H
+#define CC110X_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "periph/spi.h"
+#include "periph/gpio.h"
+#include "cc110x/cc110x-internal.h"
+
+/**
+ * @brief Struct for holding cc110x IO parameters
+ */
+typedef struct cc110x_params {
+    spi_t spi;
+    gpio_t CS;
+    gpio_t GDO0;
+    gpio_t GDO1;
+    gpio_t GDO2;
+} cc110x_params_t;
+
+typedef struct cc110x cc110x_t;
```

Comment:
you could remove this and put the name of the struct into the definition below..